usagov/usagov-2021
# Usagov 1614 breadcrumb header spacing

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1614

## Description

Reduce the spacing between the breadcrumbs and headers to match 56 px on Navigation Page
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/about-the-us and verify the spacing between the breadcrumb and the heading is 56px. Also, verify the spacing on a Spanish Nav page http://localhost/es/acerca-de-estados-unidos
- <img width="1512" alt="Screenshot 2024-06-21 at 8 48 05 AM" src="https://github.com/usagov/usagov-2021/assets/5551880/67e38841-ea3c-4802-a7b7-eb443952d1d6">


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
